README.md

```diff
@@ -118,26 +118,250 @@ services:
         command: cmd
 ```
 
-## Running pebble
+## Using pebble
 
-If pebble is installed and the `$PEBBLE` directory is set up, running it is easy:
+Pebble is invoked using `pebble <command>`. To get more information:
 
-    $ pebble run
+* To see a help summary, type `pebble -h`.
+* To see a short description of all commands, type `pebble help --all`.
+* To see details for one command, type `pebble help <command>` or `pebble <command> -h`.
 
-This will start the pebble daemon itself, and start all default services as well. Then
-other pebble commands may be used to interact with the running daemon.
+A few of the commands that need more explanation are detailed below.
 
-For example, to see any recent changes, for this or previous runs, use:
+### Running the daemon (server)
 
-    $ pebble changes
+If pebble is installed and the `$PEBBLE` directory is set up, running the daemon is easy:
 
-And start or stop a specific service with:
+```
+$ pebble run
+2022-10-26T01:18:26.904Z [pebble] Started daemon.
+2022-10-26T01:18:26.921Z [pebble] POST /v1/services 15.53132ms 202
+2022-10-26T01:18:26.921Z [pebble] Started default services with change 50.
+2022-10-26T01:18:26.936Z [pebble] Service "srv1" starting: sleep 300
+```
+
+This will start the pebble daemon itself, as well as starting all the services that
+are marked as `startup: enabled` (if you don't want that, use `--hold`). Then
+other pebble commands may be used to interact with the running daemon, for example,
+in another terminal window.
+
+To override the default configuration directory, set the `PEBBLE` environment variable when running:
+
+```
+$ PEBBLE=~/pebble pebble run
+2022-10-26T01:18:26.904Z [pebble] Started daemon.
+...
+```
+
+The daemon's service manager stores the most recent stdout and stderr from each service (using a 100KB ring buffer per service). These are accessible via the logs API or using `pebble logs`. If you want to also write service logs to Pebble's own stdout, run the daemon with `--verbose`:
```

Comment:
This seems to be going too fast into debugging. Instead, we should be talking about the model: statuses, plan, changes, etc. Otherwise this smells like a boring process runner, instead of a nicely structured service support system.

To improve this, I suggest cutting off right before the run command output, continuing through the rest of the experience, and only later mentioning the ability to see logs.

Also, before we start/stop and change the state, let's talk about reading the current state that took place right after running.

Reply:
I agree the logging stuff was going too deep (and tangential) here -- fixed. I've added a separate sub-section "Logs" at the bottom of the "Using Pebble" section. Regarding "statuses, plan, changes, etc": We talk about the plan/config in the first section as mentioned, and changes and tasks in their own section below.